src/main/java/de/tum/in/www1/artemis/service/BuildLogEntryService.java

```diff
@@ -149,7 +150,7 @@ public boolean checkIfBuildLogIsNotADuplicate(ProgrammingLanguage programmingLan
      * @param programmingLanguage the programming language
      * @return filtered build logs
      */
-    public List<BuildLogEntry> removeUnnecessaryLogsForProgrammingLanguage(List<BuildLogEntry> buildLogEntries, ProgrammingLanguage programmingLanguage) {
+    private List<BuildLogEntry> removeUnnecessaryLogs(List<BuildLogEntry> buildLogEntries, ProgrammingLanguage programmingLanguage) {
```

Comment:
Why now private ? How is this related to the PR :)

Reply:
Because the only method using this one was moved in the service.